improve AAI and RUCIO user docs (#21)
vre-hub/vre-hub.github.io: docs/auth.md

Once your account is approved, and you have confirmed your email address, you will be able to get identified towards the ESCAPE IAM service via username and password and access the VRE platform.

:::info

All the underlaying VRE services are compatible and connected with the ESCAPE IAM service. 
Once you have an ESCAPE IAM account, you will be able to interact with the different services and resources available in the platform.

:::

### OpenID token 

The ESCAPE INDIGO Iam support OpenID tokens. 

Most of the token interactions between the different component of the VRE are and should stay hidden to standard users of the platorm. Visit the [Tecnical Documentation](./tech-docs/home.md) section or the [IAM documentation](https://indigo-iam.github.io/v/v1.10.0/docs/reference/configuration/external-authentication/oidc/) for further details.

### X.509 Grid certificate    

#### *Getting the certificate*    

> Requesting a X.509 ceritiface is not straightforward and might get a bit tedious depending on your institution.

Reviewer: ```suggestion
Requesting a X.509 certificate is heavily dependent on your institution's certification authority.
```

:::tip

> Please contact your institute system administrator to understand the way to request one.

Reviewer: ```suggestion
Please contact your institute system administrator to understand how to request one.
```

# Authentication

The interaction with the VRE platform requires authentication.
It is required to register to the ESCAPE INDIGO Identity and Access Management (IAM) service to be able to access the VRE. Follow the instructions in the next sections.

### ESCAPE IAM registration

Navigate to the [ESCAPE IAM instance](https://iam-escape.cloud.cnaf.infn.it/login) and register for a new account, filling in the required details.

After the account has been approved, you will need to log in and add the `escape` group to your Groups. To do this, click on the **Home** button of the lateral bar and then click on the green **+ Add to group** button in the **Groups** panel. Enter 'escape' in the group box, and click `Add group(s)`. You can request access to various groups at the same time. You will need to wait in order for this to be manually approved by site administrators. If your request is urgent, please email the admins at _escape-cern-ops'at'cern.ch_.

Once your account is approved, and you have confirmed your email address, you will be able to get identified towards the ESCAPE IAM service via username and password and access the VRE platform.

:::note

> All the underlaying VRE services and resources are connected through the ESCAPE IAM service. 

Reviewer: ```suggestion
All the underlying VRE services and resources are connected through the ESCAPE IAM service. 
```

Once you have an account, you will be able to interact seamlessly with any of them.

:::

### OpenID token 

The ESCAPE INDIGO IAM support OpenID tokens. 

:::info

Most of the token interactions between the different component of the VRE are and should stay hidden to standard users of the platorm. 

Fur further details about these exchange, visit the [IAM documentation](https://indigo-iam.github.io/v/v1.10.0/docs/reference/configuration/external-authentication/oidc/) or navigate through the different sections of the [VRE Tecnical Documentation](./tech-docs/home.md).

:::

### X.509 Grid certificate    

#### *Getting the certificate*    

Requesting a X.509 ceritiface is not straightforward and might get a bit tedious depending on your institution.

:::tip

> Please contact your institute system administrator to understand the best way to request one.

Reviewer: ```suggestion
Please contact your institute system administrator to understand how to request one.
```
